source/_components/counter.markdown

```diff
@@ -24,18 +24,45 @@ counter:
     step: 1
 ```
 
-Configuration variables:
+{% configuration %}
+# 'alias' should be replaced by the user for their actual value.
+"[alias]":
+  description: Alias for the counter. Multiple entries are allowed.
+  required: true
+  type: map
+  keys:
+    name:
+      description: Friendly name of the counter.
+      required: false
+      type: string
+    initial:
+      description: Initial value when Home Assistant starts or the counter is reset.
+      required: false
+      type: integer
+      default: 0
+    restore:
+      description: Try to restore the last known value when Home Assistant starts.
+      required: false
+      type: boolean
+      default: True
+    step:
+      description: Incremental/step value for the counter.
+      required: false
+      type: integer
+      default: 1
+    icon:
+      description: Icon to display for the counter.
+      required: false
+      type: icon
```

Comment:
`icon` type?

Reply:
That's what I found in various other pages (input_boolean, input_number, input_select)

Reply:
`icon:` is now handled like `device_class:` (#6495). Thanks for pointing it out @WofWca.